test/regression/neurofeedback.py

```diff
@@ -109,23 +110,24 @@ class RunParams:
 
 
 def run_cmd(params: RunParams) -> str:
+    logging.info(f"running job: {params}")
     proc = subprocess.Popen(params.cmd, cwd=params.cwd, env=params.env, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
     out, err = proc.communicate()
     logging.info(f"job finished: {params}")
+    logging.debug(f"stdout: {out}")
+    logging.debug(f"stderr: {err}")
     return out
 
 
-def regression_test():
-    sif_path = Path(os.getcwd()) / "bin" / "murfi.sif"
+def regression_test(no_image, include_pre_header):
+    base_exec_path = Path(os.getcwd()) / "bin" / "murfi"
+    base_cmd = [] if no_image else ["singularity", "exec", base_exec_path.with_suffix(".sif")]
 
+    murfi_exec = [base_exec_path] if no_image else base_cmd + ["murfi"]
     murfi_params = RunParams(
-        cmd=[
-            "singularity",
-            "exec",
-            sif_path,
-            "murfi",
+        cmd=murfi_exec + [
             "-f",
-            "scripts/neurofeedback.xml",
+            "scripts/neurofeedback-preheader.xml" if include_pre_header else "scripts/neurofeedback.xml",
```

Comment:
I don't think these xml files exist in the repo.

Reply:
yup, that's right. they are in the example data that the github workflow downloads